sormas-app/app/src/main/java/de/symeda/sormas/app/person/SelectOrCreatePersonDialog.java

```diff
@@ -193,6 +218,11 @@ public void onPropertyChanged(Observable observable, int i) {
 				if (getSelectedPerson() == null) {
 					btnCreate.setVisibility(View.VISIBLE);
 					btnSelect.setVisibility(View.GONE);
+
+					//TODO: View based on tag and the item will be set to false
```

Comment:
Is this TODO still valid? If yes, why? If no, please remove the comment

Reply:
No, only forgot there :) I'll delete it